frontend/src/metabase/browse/components/ModelsTable.tsx

```diff
@@ -8,6 +8,7 @@ import { push } from "react-router-redux";
 import { t } from "ttag";
 
 import { getCollectionName } from "metabase/collections/utils";
+import { EllipsifiedPath } from "metabase/common/components/EllipsifiedPath/EllipsifiedPath";
```

Comment:
we probably want an index.ts file so we can import as `metabase/common/components/EllipsifiedPath`